openedx/core/djangoapps/enrollments/paginators.py

```diff
@@ -11,3 +11,6 @@ class CourseEnrollmentsApiListPagination(CursorPagination):
     Paginator for the Course enrollments list API.
     """
     page_size = 100
+    page_size_query_param = 'page_size'
+    max_page_size = 100
+    page_query_param = 'page'
```

Comment:
Specifying the `?page_size=2&page=2` does not seem to work for me.

Reply:
I tried [with page_size=1&page=2 ](http://localhost:18000/api/enrollment/v1/enrollments?page_size=1&page=2), seems to be working for me. Am I missing something?

![image](https://user-images.githubusercontent.com/7670449/213161294-dfc6b5d2-8232-4d16-83c3-e7fcafd2a147.png)

Reply:
I tried [with page_size=1&page=2 ](http://localhost:18000/api/enrollment/v1/enrollments?page_size=1&page=2), seems to be working for me. Am I missing something?

![image](https://user-images.githubusercontent.com/7670449/213161294-dfc6b5d2-8232-4d16-83c3-e7fcafd2a147.png)